containers/runtime/driver/Dockerfile

```diff
@@ -87,4 +87,4 @@ ENV QPS_WORKERS_FILE=""
 ENV SCENARIOS_FILE="/src/driver/example.json"
 ENV BQ_RESULT_TABLE=""
 
-CMD ["/src/driver/run.sh"]
+CMD ["bash", "-c", "timeout --kill-after=$KILL_AFTER $POD_TIMEOUT /src/driver/run.sh"]
```

Comment:
Would it work if you would separate every argument here, "bash", "-c", "timeout", "--kill-after=${KILL_AFTER}" and so on? I would even like to see if we can have "" around the variables, but that may be too much.

Reply:
I will try that

Reply:
I have tried to use:
```
CMD ["bash", "-c", "timeout", "--kill-after=${KILL_AFTER}", "${POD_TIMEOUT}", " /src/driver/run.sh"]
```
Error with:
```
Try 'timeout --help' for more information.
```
Seems like the argument for `bash -c` has to be in a single string. 

I put the `{}` for the variable, it doesn't allow ""  to wrap around the variables.